### PR TITLE
Fix #6043: Add 1.44 translations.

### LINKED ISSUE #6043: Add 1.44 translations

### PR DESCRIPTION
**Pull request by iccub**
## Summary of Changes

This pull request fixes #6043 

## Submitter Checklist:

- [x] *Unit Tests* are updated to cover new or changed functionality
- [x] User-facing strings use `NSLocalizableString()`
- [x] New or updated UI has been tested across:
  - [x] Light & dark mode
  - [x] Different size classes (iPhone, landscape, iPad)
  - [x] Different dynamic type sizes

## Test Plan:

Go to app settings.
Verify that text for bottom/top toolbar settings is updated.

## Reviewer Checklist:

- [x] Issues include necessary QA labels:
  - `QA/(Yes|No)`
  - `bug` / `enhancement`
- [x] Necessary [security reviews](https://github.com/brave/security/issues/new/choose) have taken place.
- [x] Adequate unit test coverage exists to prevent regressions.
- [x] Adequate test plan exists for QA to validate (if applicable).
- [x] Issue and pull request is assigned to a milestone (should happen at merge time).

### CODE REVIEW

File: App/l10n/Resources/tr.lproj/BraveShared.strings [hunk 1]
```diff
@@ -2551,6 +2596,15 @@
 /* Title for settings menu item */
 "SettingsMenuItem" = "Ayarlar";
 
+/* An option for the URL bar position. (Top Bar and Bottom Bar). Refers to the location of the url bar */
+"SettingsTabsOptionBottomBar" = "Alt Çubuk";
```
Comment (soner-yuksel) [inline App/l10n/Resources/tr.lproj/BraveShared.strings]: Alt Sekme Çubuğu

File: App/l10n/Resources/tr.lproj/BraveShared.strings [hunk 2]
```diff
@@ -2551,6 +2596,15 @@
...
+
+/* An option for the URL bar position. (Top Bar and Bottom Bar). Refers to the location of the url bar */
+"SettingsTabsOptionTopBar" = "Üst çubuk";
```
Comment (soner-yuksel) [inline App/l10n/Resources/tr.lproj/BraveShared.strings]: Üst Sekme Çubuğu

File: App/l10n/Resources/tr.lproj/BraveWallet.strings [hunk 3]
```diff
@@ -334,6 +331,27 @@
...
+"wallet.dappsSettingsConnectedSitesSectionTitle" = "Bağlı DApps";
+
+/* The title of the section that will display settings for default wallet and dapp provider api access permission */
+"wallet.dappsSettingsGeneralSectionTitle" = "Genel";
+
+/* The navigation title of the settings screen for each Dapp. '%@' will be relpaced by the name of the coin type. For example, the title could 'Ethereum Settings', 'Solana Settings' etc. */
+"wallet.dappsSettingsNavTitle" = "%@ Ayarları";
+
+/* The warning message in a confirmation dialog when attempting to remove all wallet connection to a dapp. */
+"wallet.dappsSettingsRemoveAllWarning" = "Bunu yapmak %1$d %2$@ için tüm Cüzdan bağlantısı izinlerini kaldırır";
```
Comment (soner-yuksel) [inline App/l10n/Resources/tr.lproj/BraveWallet.strings]: No need for the second parameter here `%2$@`. Making website plural in Turkish will be a grammatical error.

Change it as `Bunu yapmak %1$d websitesi için tüm Cüzdan bağlantısı izinlerini kaldırır`

File: App/l10n/Resources/zh.lproj/BraveWallet.strings [hunk 4]
```diff
@@ -334,6 +331,27 @@
...
+"wallet.dappsSettingsConnectedSitesSectionTitle" = "关联的 DApps";
+
+/* The title of the section that will display settings for default wallet and dapp provider api access permission */
+"wallet.dappsSettingsGeneralSectionTitle" = "通用";
+
+/* The navigation title of the settings screen for each Dapp. '%@' will be relpaced by the name of the coin type. For example, the title could 'Ethereum Settings', 'Solana Settings' etc. */
+"wallet.dappsSettingsNavTitle" = "%@ 设置";
+
+/* The warning message in a confirmation dialog when attempting to remove all wallet connection to a dapp. */
+"wallet.dappsSettingsRemoveAllWarning" = "这将删除 %1$d %2$@ 的所有电子钱包关联权限";
```
Comment (nuo-xu) [inline App/l10n/Resources/zh.lproj/BraveWallet.strings]: "这将删除 %1$d个 %2$@ 的所有电子钱包关联权限"

File: App/l10n/Resources/zh-TW.lproj/BraveWallet.strings [hunk 5]
```diff
@@ -334,6 +331,27 @@
...
+"wallet.dappsSettingsConnectedSitesSectionTitle" = "連結的 DApp";
+
+/* The title of the section that will display settings for default wallet and dapp provider api access permission */
+"wallet.dappsSettingsGeneralSectionTitle" = "一般";
+
+/* The navigation title of the settings screen for each Dapp. '%@' will be relpaced by the name of the coin type. For example, the title could 'Ethereum Settings', 'Solana Settings' etc. */
+"wallet.dappsSettingsNavTitle" = "%@ 設定";
+
+/* The warning message in a confirmation dialog when attempting to remove all wallet connection to a dapp. */
+"wallet.dappsSettingsRemoveAllWarning" = "這會移除 %1$d %2$@ 的所有錢包連結權限";
```
Comment (nuo-xu) [inline App/l10n/Resources/zh-TW.lproj/BraveWallet.strings]: "這會移除 %1$d個 %2$@ 的所有錢包連結權限"

**Review by nuo-xu — CHANGES_REQUESTED** [review]

**Review by soner-yuksel — APPROVED** [review]

**Review by nuo-xu — APPROVED** [review]

**Pull request merged**